tests/mbt/model/ccv_model.qnt

```diff
@@ -517,9 +541,10 @@ module ccv_model {
                     assert(currentState.providerState.chainState.votingPowerHistory == List(InitialValidatorSet.put("node1", 150), InitialValidatorSet)),
                     // change voting power on provider again
                     VotingPowerChange("node1", 50).then(
-                        // end another block
-                        EndAndBeginBlockForProvider(1 * Second, Set(), Set())
-                    ).then(
+                        // end the epoch
+                        EndProviderEpoch(3 * Second, Set(), Set())
```

Comment:
Why do we have 3 seconds here instead of 1 as is done in `EndProviderEpoch` below?

Reply:
"Historical reasons", i.e. a relic from trying things while debugging :)
good catch, adjusted this!